libs/blocks/merch-card/merch-card.js

```diff
@@ -3,12 +3,15 @@ import { getConfig, createTag } from '../../utils/utils.js';
 import { getMetadata } from '../section-metadata/section-metadata.js';
 import { processTrackingLabels } from '../../martech/attributes.js';
 import { replaceKey } from '../../features/placeholders.js';
+import { decorateMnemonicList } from '../mnemonic-list/mnemonic-list.js';
```

Comment:
`decorateMnemonicList` is not needed for majority of the cards, hence should be loaded dynamically and conditionally.

Reply:
Fixed, now dynamically imported.